src/openrct2/ride/RideData.h

```diff
@@ -538,7 +559,10 @@ constexpr RideTypeDescriptor DummyRTD =
     .Flags = 0,
     .RideModes = EnumsToFlags(RideMode::ContinuousCircuit),
     .DefaultMode = RideMode::ContinuousCircuit,
-    .OperatingSettings = { 0, 0, 0, 0, 0, 0 },
+    .OperatingSettings = { 0, 0, 1 },
+    .TrackSpeedSettings = { 30, 30 },
+    .BoosterSettings = {0, 0, 12},
+    .LegacyBoosterSettings = {0, 0, 2},
```

Comment:
You can omit the values here as you already specified them on the structs themselves.
```suggestion
    .OperatingSettings = {},
    .TrackSpeedSettings = {},
    .BoosterSettings = {},
    .LegacyBoosterSettings = {},
```